app/dashboard/helpers.py

```diff
@@ -670,6 +670,19 @@ def record_user_action(event_name, old_bounty, new_bounty):
             })
 
 
+def is_lowball_bounty(bounty_value_usdt):
+    """Determine if a bounty value is less than a threshold
+
+    Args:
+      bounty_value_usdt (Decimal): The value of the bounty
+
+    Returns:
+      bool: True if bounty value is less than the threshold
+
+    """
+    return bounty_value_usdt < 10.00
```

Comment:
Is there a reason why it's hardcoded "10.00"? Could we put it somewhere more obvious, like the settings.py ?